### PR TITLE
Added Logarithm Calculator

### LINKED ISSUE #26: Logarithm Calculator
### Prerequisites

- [X] I am an Open Source Contributor
- [X] I checked to make sure that this issue has not already been filed

### Description

@Rakesh9100 I want to add new feature which calculates the log of the number taken as input. Please assign this issue to me

### Screenshots

_No response_

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/CalcDiverse/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.

### PR DESCRIPTION
**Pull request by soumyaarya**
# Fixes Issue🛠️

Closes #26 

# Description👨‍💻 

Added Log Calculator

# Type of change📄

- [] New feature (non-breaking change which adds functionality)

# How this has been tested✅

locally

# Checklist✅ 

- [] My code follows the style guidelines of this project
- [] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have added demonstration in the form of GIF/video file
- [] I am an Open Source Contributor

# Screenshots/GIF📷
Screenshot (93)
Screenshot (92)

### CODE REVIEW

**Comment by Shreya111111** [comment]
@soumyaarya the site cannot be reached plz check your code and index.html

**Comment by soumyaarya** [comment]
I have updated index.html pls recheck @Shreya111111

**Comment by Rakesh9100** [comment]
Main `index.html` page is not updated properly by your end.

Just add your calc section and no any other changes @soumyaarya

**Comment by soumyaarya** [comment]
okay @Rakesh9100  I will change it

**Comment by soumyaarya** [comment]
@Rakesh9100 I have updated main index.html pls recheck it

File: index.html [hunk 1]
```diff
@@ -310,6 +322,7 @@ <h3>Calcualte the different results of a triangle based on input sides</h3>
     <div class="contributors">
         <h1>Our Valuable Contributors</h1>
         <div id="contributor"></div>
+ main
```
Comment (Rakesh9100) [inline index.html:325]: What is this `main`, pls delete this @soumyaarya

**Comment by Rakesh9100** [comment]
Also change your folder name to `Logarithm-Calculator` and make the changes accordingly @soumyaarya

**Comment by Rakesh9100** [comment]
Merged!! @soumyaarya

**Pull request merged**